templates/win2k12r2.tpl.yaml

```diff
@@ -39,6 +39,13 @@ metadata:
           "rule": "integer",
           "message": "This VM requires more memory.",
           "min": {{ lookup('osinfo', osinfoname)["minimum_resources.0.ram"] }}
+        }, {
+          "name": "windows-virtio-bus",
+          "path": "jsonpath::.spec.domain.devices.disks[*].disk.bus",
```

Comment:
What about exposing two rules? One enforcing that tells the user that valid options are virtio and sata and the other like you wrote - just warning.